src/parse.ts

```diff
@@ -410,6 +410,8 @@ function parseStringSchema(details: StringDescribe, settings: Settings, rootSche
 
 function parseArray(details: ArrayDescribe, settings: Settings): TypeContent | undefined {
   const { interfaceOrTypeName, jsDoc } = getCommonDetails(details, settings);
+  // @ts-expect-error Idk why but the sparse flag is not in the type
+  const isSparse = details.flags?.sparse;
```

Comment:
There is a custom types file for this you can add it here https://github.com/mrjono1/joi-to-typescript/blob/master/src/joiDescribeTypes.ts#L10

Reply:
@mrjono1 - Done, thanks for the tip